.gitmodules

```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/SpecularL2/blockscout
 [submodule "services/el_clients/go-ethereum"]
 	path = services/el_clients/go-ethereum
-	url = https://github.com/specularL2/go-ethereum
+	url = https://github.com/SpecularL2/go-ethereum.git
```

Comment:
why change this?

Reply:
already overwritten - I changed this to ssh path and IMHO we should do same with the rest of submodules, since (at least for me) HTTPS return:
```
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/specularL2/go-ethereum/'
```